persist/fs/seek.go

```diff
@@ -78,6 +78,17 @@ func NewSeeker(filePathPrefix string, bufferSize int, bytesPool pool.BytesPool)
 	}
 }
 
+// GetFileIds retrieves all the identifiers present in the configured file
```

Comment:
On interesting, golint should realize that it's an interface method implementation... Hm.  Was this before it was added to the `Seeker` interface perhaps?  Or still complains about it now if this comment is removed?